nginx.conf

```diff
@@ -79,6 +79,8 @@ http {
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/cookbook/timeseries/$                         $scheme://$http_host/docs/$1/subgraphs/best-practices/timeseries/                   permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/subgraphs/cookbook/timeseries/$               $scheme://$http_host/docs/$1/subgraphs/best-practices/timeseries/                   permanent;
         rewrite ^/docs/([a-zA-Z][a-zA-Z])/cookbook/transfer-to-the-graph/$              $scheme://$http_host/docs/$1/subgraphs/cookbook/transfer-to-the-graph/              permanent;
+        rewrite ^/docs/([a-zA-Z][a-zA-Z])/subgraphs/cookbook/subgraph-uncrashable$      $scheme://$http_host/docs/$1/subgraphs/developing/creating/install-the-cli/         permanent;
```

Comment:
Why handle this one separately, when it looks like it got moved to `guides` like the other ones?